docs/reference/remap/parse_aws_vpc_flow.cue

```diff
@@ -0,0 +1,86 @@
+package metadata
+
+remap: functions: parse_aws_vpc_flow: {
+	arguments: [
+		{
+			name:        "value"
+			description: "VPC Flow Log."
+			required:    true
+			type: ["string"]
+		},
+		{
+			name:        "format"
+			description: "VPC Flow Log format."
+			required:    false
+			type: ["string"]
+		},
+	]
+	return: ["map"]
+	category: "parse"
+	description: #"""
+		Parses a [VPC Flow Logs]\(urls.aws_vpc_flow_logs\) into it's constituent components.
+		"""#
+	examples: [
+		{
+			title: "Success"
+			input: {
+				log: #"2 123456789010 eni-1235b8ca123456789 - - - - - - - 1431280876 1431280934 - NODATA"#
+			}
+			source: #"""
+				.parsed = parse_aws_vpc_flow(.log)
+				"""#
+			output: {
+				log: #"2 123456789010 eni-1235b8ca123456789 - - - - - - - 1431280876 1431280934 - NODATA"#
+				parsed: {
+					"version":      2
+					"account_id":   123456789010
+					"interface_id": "eni-1235b8ca123456789"
+					"srcaddr":      null
+					"dstaddr":      null
+					"srcport":      null
+					"dstport":      null
+					"protocol":     null
+					"packets":      null
+					"bytes":        null
+					"start":        1431280876
+					"end":          1431280934
+					"action":       null
+					"log_status":   "NODATA"
+				}
```

Comment:
I'm not sure here, do we need to add the key to the map if the value is `null`? @JeanMertz

Reply:
I don't have a good answer.

My initial response would be "no", but if AWS does it, then we probably should as well, as that's what people expect from their output, maybe? Maybe our resident AWS expert @jszwedko has some thoughts on this.

Reply:
I think people will expect a consistent set of fields to be defined when parsing these logs so setting to `null` makes sense.